src/app/models/user-info.model.ts

```diff
@@ -0,0 +1,12 @@
+export class UserInfoModel {
```

Comment:
There is already userInfo model

Reply:
that one is a little different, without role inside

Reply:
Then add what you need to existing model and don`t create a new one